.gitignore

```diff
@@ -47,3 +47,4 @@ doc/plugins_guide.rst
 doc/CHANGELOG*.md
 doc/CHANGELOG*.rst
 doc/changelogs.rst
+.idea
```

Comment:
? what is it ?

Reply:
The .idea folder in the solution root contains IntelliJ’s project specific settings files.